db/1.9/modules/ROOT/edits/database-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,47 @@
+= Database Connector Troubleshooting - Mule 4
+
+To troubleshoot Anypoint Connector for Database (Database Connector), review the following causes and possible solutions.
+
+== Pooling Profiles Connections Making the Flow Hang
+Using a pooling profile that provides fewer connections than what the flow uses can make the entire flow hang.
+
+=== Solution
+One solution is to increase the size of the pool. However, sometimes that's not possible, so another solution is to use the VM Connector to shorten the life of the connection to be the same as the life of a VM message. This way, every call to the *Stored procedure* operation is executed, its result is consumed, and the connection associated with the operation is closed and returned to the pool.
+
+The following example shows how to use pooling profiles with the *Stored procedure* operation and VM Connector. When the `foreach` element reaches the eleventh iteration the *Stored procedure* operation won't hang since the pool is able to provide a connection:
```

Comment:
```suggestion
The following example shows how to use pooling profiles with the *Stored procedure* operation and VM Connector. In this example, when the `foreach` element reaches the eleventh iteration, the *Stored procedure* operation doesn't hang, because the pool is able to provide a connection:
```